_i18n/ar/resources/moneropedia/networkupgrade.md

```diff
@@ -0,0 +1,8 @@
+---
+terms: ["networkupgrade"]
+summary: Network Upgrades (hard forks) are non-contentious and no new coin will be created. We have multiple blog posts we could link to.
```

Comment:
This should be a summary that explains in few words what the meaning of "network-upgrade" is. We can add more details later in the entry itself. Also, the last phrase can be dropped.

Reply:
This is still unchanged